cartoframes/viz/source.py

```diff
@@ -130,7 +130,9 @@ def get_credentials(self):
         credentials = self.dataset.credentials
         if credentials:
             return {
-                'username': credentials.username,
+                # VL requires !username but CF allows passing only the base_url.
```

Comment:
I would rephrase this, using CARTO VL instead of VL, CARTOframes instead of CF, etc. I know this is internal but I'd try to keep it consistent.